Clarify docs for pytest.main default behavior (#11187)
pytest-dev/pytest: doc/en/how-to/usage.rst

this acts as if you would call "pytest" from the command line.
It will not raise :class:`SystemExit` but return the :ref:`exit code <exit-codes>` instead.

> Be careful though, by default (when ``None`` is passed) ``main`` will read arguments
> from the command line, which may not be desirable.
> You can pass in options and arguments explicitly:

Reviewer: ```suggestion
If you don't pass it any arguments, ``main`` reads the arguments from the command line arguments of the process (:data:`sys.argv`), which may be undesirable.
You can pass in options and arguments explicitly:
```
